channels/ir.m3u

```diff
@@ -17,6 +17,10 @@ https://live.al-zahratv.com/live/playlist2/index.m3u8
 https://nl.live21.ir/hls2/AMG_mid.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/UFV7YVy.jpg" group-title="",Andisheh TV
 http://livestream.5centscdn.com/cls090117/49ead3a7c6432c5358a086d078110c16.sdp/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Kurdish" tvg-logo="https://avaentertainment.net/img/logos/m-logo.png" group-title="Entertainment",AVA TV
+https://wowzaprod219-i.akamaihd.net/hls/live/1003949/c9aa559e/c9aa559e_1_2160000/chunklist.m3u8
```

Comment:
It's an Iraqi channel (https://www.lyngsat.com/tvchannels/iq/Ava-Entertainment.html)

Reply:
This channels is targeting Kurdish speaking Iranians. Kurds are in Iran, Iraq and Turkey. I am just trying to add some channels for Iranians that their 1st language is not Persian.

Reply:
If you specify right language in the `tvg-language` attribute, the channel will automatically be included in the playlist corresponding to that language: https://github.com/iptv-org/iptv#playlists-by-language

While these files are based only on the country from which the channel is broadcast. More details here: https://github.com/iptv-org/iptv/blob/master/.github/CONTRIBUTING.md#sort-channels-by-country

Reply:
80% of channels in ir.m3u are not broadcasting from Iran. There are six live feed cameras named Karbala which is in Iraq and has nothing to do with Iran. Maybe whoever added those channels thought that some Moslem Iranians might be interested in a holy site live feed. There are many languages spoken in Iran including Persian (Farsi), Arabic, Azerbaijani and Kurdish. I think this Kurdish channel has a dialect spoken in Iran.

Reply:
If you think that some channels are in this playlist by mistake, you feel free to fix it. Just please read this guide first: https://github.com/iptv-org/iptv/blob/master/.github/CONTRIBUTING.md

If you have any suggestions on how to sort the channels, you'd better create a separate [issue](https://github.com/iptv-org/iptv/issues) so that others can participate in the discussion as well.

Reply:
I think sorting the channels based on the name of the country or language is good enough but Iran's case is different. A lot of these providers will not be able to operate in Iran due to the type of programing that they offer but they are targeting Iranians inside and outside of Iran. So I think it is appropriate to keep all these channels under the same playlist regardless of the language or where they are being broadcasted from as long as the target audience are Iranians. So far I have added channels and only removed one channel due to a loop stream. I do not think that I should remove a working channel from this playlist as long as is meant for Iranians. 
I think these channels belong to this playlist but if you do not think so, please feel free to close this request.